phone/src/apps/settings/components/WallpaperModal.tsx

```diff
@@ -37,11 +37,8 @@ export default function WallpaperModal() {
   };
 
   const handleNewWallpaper = () => {
-    handleSettingChange('customWallpaper', value);
-  };
-
-  const handleRemoveWallpaper = () => {
-    handleSettingChange('customWallpaper', '');
+    handleSettingChange('wallpaper', { label: 'Custom Wallpaper', value });
```

Comment:
Here, this isn't being set as translation key, but it is being referenced as a translation key in Phone.tsx.

Reply:
xd moment